Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -368,6 +368,45 @@ def download_artifact(url, dir, pr)
     end
   end
 
+  def pr_check_conflicts(name, tap_remote_repo, pr)
+    hash_template = proc.new { |h, k| h[k] = [] }
+    long_build_pr_files = GitHub.search_issues(
+      "org:#{name}", repo: tap_remote_repo, state: "open", label: "\"no long build conflict\""
+    ).each_with_object(Hash.new(hash_template)) do |long_build_pr, hash|
+      number = long_build_pr["number"]
+      GitHub.get_pull_request_changed_files(name, tap_remote_repo, number).each do |file|
+        key = file["filename"]
+        hash[key] << number
+      end
```

Comment:
```suggestion
    long_build_pr_files = GitHub.search_issues(
      "org:#{name}", repo: tap_remote_repo, state: "open", label: 
      '"no long build conflict"'
    ).each_with_object({}) do |long_build_pr, hash|
      number = long_build_pr["number"]
      GitHub.get_pull_request_changed_files(name, tap_remote_repo, number).each do |file|
        key = file["filename"]
        hash[key] ||= []
        hash[key] << number
      end
```

I think this is a little more readable and is no more lines of code.

Reply:
Oops, this reverses my earlier suggestion 😅 @iMichka can decide whichever is preferred.

Reply:
Yeh, I don't mind too much either way, just think this way is a little more intuitive to non-Rubyists.